invenio_oauthclient/version.py

```diff
@@ -28,4 +28,4 @@
 and parsed by ``setup.py``.
 """
 
-__version__ = '1.0.0a14.dev20170511'
+__version__ = '1.0.0a14.dev20170802'
```

Comment:
> One more small thingy could you please keep the commit subject line to 50 chars? :P

I'm all out of ideas for clean, short and unambiguous title. Do you have suggestions?